### PR TITLE
47117_db_migration_model_creation

### PR DESCRIPTION
**Pull request by manazirahsan**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ source 'https://rubygems.org'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.5'
 # Use sqlite3 as the database for Active Record
-#gem 'sqlite3'
-gem 'mysql2'
+gem 'sqlite3'
```
Comment (noor-framgia) [inline Gemfile]: aren't we using mysql?

File: .idea/.generators [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/.generators]: do not track this file. Remove

File: .idea/.name [hunk 3]
```diff
@@ -0,0 +1 @@
+tms_41
```
Comment (shakil-fg) [inline .idea/.name]: do not track this file. Remove

File: .idea/.rakeTasks [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/.rakeTasks]: do not track this file. Remove

File: .idea/misc.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/misc.xml]: do not track this file. Remove

File: .idea/modules.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/modules.xml]: do not track this file. Remove

File: .idea/runConfigurations/Development__tms_41.xml [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (shakil-fg) [inline .idea/runConfigurations/Development__tms_41.xml]: do not track this file. Remove

File: .idea/runConfigurations/Production__tms_41.xml [hunk 8]
```diff
@@ -0,0 +1,28 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (shakil-fg) [inline .idea/runConfigurations/Production__tms_41.xml]: do not track this file. Remove

File: .idea/runConfigurations/test__tms_41.xml [hunk 9]
```diff
@@ -0,0 +1,28 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (shakil-fg) [inline .idea/runConfigurations/test__tms_41.xml]: do not track this file. Remove

File: .idea/tms_41.iml [hunk 10]
```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/tms_41.iml]: do not track this file. Remove

File: .idea/vcs.xml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/vcs.xml]: do not track this file. Remove

File: .idea/workspace.xml [hunk 12]
```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shakil-fg) [inline .idea/workspace.xml]: do not track this file. Remove

File: Gemfile (hunk 1 above)
Comment (shakil-fg) [inline Gemfile]: Do not use sqlite, It was requested during last project

File: config/database.yml [hunk 13]
```diff
@@ -0,0 +1,44 @@
+#common: &common
```
Comment (shakil-fg) [inline config/database.yml]: Do not use sqlite, It was requested during last project

File: config/database.yml.sample [hunk 14]
```diff
@@ -1,24 +0,0 @@
-common: &common
```
Comment (shakil-fg) [inline config/database.yml.sample]: Do not remove sample file

File: config/database.yml (hunk 13 above)
Comment (shakil-fg) [inline config/database.yml]: Do not track this file. Remove from tracking list

**Comment by shakil-fg** [comment]
OK

**Comment by noor-framgia** [comment]
ok.

**Comment by mdmahmudurrahman** [comment]
OK.

**Pull request merged**